admin/data/sql/migrations/20200318120158.undo.update-check-add-created-by-user-id.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE [mtc_admin].[check]
+DROP CONSTRAINT [FK_check_createdBy_userId];
+ALTER TABLE [mtc_admin].[check]
```

Comment:
Is this okay like this?  Or should the two ALTERs go in separate files?

Reply:
Looks ok to me. they can happen in a single transaction.